include/ControllerPose.h

```diff
@@ -7,10 +7,18 @@
 class ControllerPose {
  public:
   ControllerPose(vr::ETrackedControllerRole shadowDeviceOfRole, std::string thisDeviceManufacturer,
-                 VRPoseConfiguration_t poseConfiguration);
+                 VRPoseConfiguration_t poseConfiguration, bool isRightHand);
```

Comment:
is there a need for `isRightHand`? Handedness should be able to be figured out through `shadowDeviceOfRole`